components/monitoring/logging/staging/stone-stage-p01/configure-logforwarder-compression-patch.yaml

```diff
@@ -1,5 +1,9 @@
 ---
 - op: add
-  path: /spec/tuning
-  value: 
+  path: /spec/outputs/0/tuning
```

Comment:
you need to regenerate the deploy.yaml

Reply:
oh, sorry about the confusion... I was hoping from one PR to another and I mixed up the components. This PR is in monitoring, there is not generations needed here... (face_palm)